draft-ietf-core-cf-reg-update.md

```diff
@@ -38,53 +38,56 @@ normative:
 For the FCFS portion of the registry, these rules do not involve the Designated Expert (DE) and are managed solely by IANA personnel to finalize the registration.
 
 Unfortunately, the instructions do not explicitly require checking that the combination of content-type (i.e., media type with optional parameters) and content coding associated with the requested CoAP Content-Format is semantically valid.
-This task is generally non-trivial, requiring knowledge from multiple documents and technologies, which is not a task to demand solely from the registrar.
+This task is generally non-trivial, requires knowledge from multiple documents and technologies, and should not be solely demanded from the registrar.
 This lack of guidance may engender confusion in both the registering party and the registrar, and has already led to erroneous registrations.
 
-In {{iana}}, this document updates {{-coap}} by modifying the registration procedures for the "CoAP Content-Formats" registry regarding its IETF Review or IESG Approval portion as well as its FCFS portion, to mitigate the risk of unintentional or malicious errors.
+In {{iana}}, this document updates {{-coap}} by modifying the registration procedures for the "CoAP Content-Formats" registry to mitigate the risk of unintentional or malicious errors.
+These updates amend the different portions of the registry, introduce a review procedure to be performed for most portions of the registry, and allow the registration of temporary Content-Format identifiers for certain portions of the registry.
+This document also introduces a new column, "Media Type", to the registry.
 
 # Conventions and Definitions
 
 {::boilerplate bcp14-tagged-bcp14}
 
-This document uses the terms "media type", "content coding", "content-type" and "content format" as defined in {{Section 2 of -senml-cf}}.
+This document uses the terms "media type", "content coding", "content-type", and "content format" as defined in {{Section 2 of -senml-cf}}.
 
 # Examples for Erroneous Registrations
 
-This section contains a few examples of registration requests for a CoAP Content-Format with identifier 64999 in the FCFS space that must not be allowed to succeed.
+This section contains examples of registration requests for a CoAP Content-Format with identifier 64999 in the FCFS portion of the "CoAP Content-Formats" registry, which must not be allowed to succeed.
 
-The following considerations also apply to alternative examples where, for the same combination of content type and content coding, a registration was requested for a CoAP Content-Format with identifier in the IETF Review or IESG Approval space. That is, such registrations must not be allowed to succeed.
+The following considerations also apply to alternative examples where, for the same combination of content type and content coding, a registration was requested for a CoAP Content-Format with identifier in the IETF Review or IESG Approval portion of the registry.
```

Comment:
```suggestion
The following considerations also apply for a registrations requested for a CoAP Content-Format with an identifier in the IETF Review or IESG Approval portion of the registry.
```

Reply:
Sorry, what are the "considerations" here that the sentence talks about? Shouldn't it be "...the following examples..." ?

Or something like:

```
For each of the following example registration requests, a similar example can be constructed where the requested registration is for the CoAP Content-Format identifier 9999 in the IETF Review or IESG Approval portion of the registry.
```

Isn't that more clear and concrete (with number 9999) ?

Reply:
Trying to merge @cabo and @EskoDijk suggestions into a mad Frankenstein:

> _For each of the following example registration requests, one can create a similar instance where the requested registration is for a CoAP Content-Format identifier within the "IETF Review" or "IESG Approval" portion of the registry._

WDYT?

Reply:
done in [3d11d2b](https://github.com/core-wg/cf-reg-update/pull/40/commits/3d11d2b7bcd22c3a978a96225772eae8a1544067)